src/index.js

```diff
@@ -1,8 +1,9 @@
-import { Component, ComponentManager } from './core';
-import * as plugins from './plugins';
+import componentProvider from './componentProvider';
+import componentLoader from './componentLoader';
+import domContentLoaded from './domContentLoaded';
 
 export {
-  ComponentManager,
-  Component,
-  plugins,
+  componentProvider,
+  componentLoader,
+  domContentLoaded,
```

Comment:
Any reason for exporting the `domContentLoaded` function?

Reply:
I imagined it being useful. That said, it's not documented and I don't think it's worth documenting, so maybe I just remove it.